### DISCUSSION TITLE
Persistent notifications don't seem to work on wearable devices

### DISCUSSION THREAD

**Discussion by ghost** [iSoron/uhabits#995] [Help & FAQ]
Recently I noticed that I was no longer getting Habits notifications on my FitBit. I believe it was because I enabled "persistent notifications". I am not certain, but I've read that wearable devices don't support persistent notifications.

Just wanted to post here to see if it maybe helpful for someone else who encounters the same issue. If that's the actual cause, perhaps adding a blurb about "ymmv using persistent notifications on wearable devices" in the app settings would be helpful

**Comment by baincd** [comment]
Thank you!  I was having the same issue - Habit notifications were not being sent to my FitBit.  I disabled sticky notifications, and it works perfectly.  Thank you for sharing!!
